rpc/gnmi/gnmi-history.md

```diff
@@ -0,0 +1,47 @@
+# gNMI History Extension
```

Comment:
(nit): Can you 80ch wrap the doc by any chance? It just makes things easier to review in the future.

Reply:
Done.